modules/os2dagsorden/os2dagsorden_access_helper/inc/os2dagsorden_access_helper.permissions.inc

```diff
@@ -84,7 +84,7 @@ function os2dagsorden_access_helper_check_user_can_see_meeting($meeting, $commit
 
     // Checking if is meeting in hidden agendas list.
     $hidden_agendas = explode(',', variable_get('os2dagsorden_hidden_agenda_committee'));
-    if (strpos(strtolower($meeting->field_os2web_meetings_type['und'][0]['value']), "dagsorden") !== FALSE) {
+    if (strpos(strtolower($meeting->field_os2web_meetings_type['und'][0]['value']), "dagsorden") === 0) {
```

Comment:
i suggest simplifying it by adding the following change:

`if ($meeting->field_os2web_meetings_type['und'][0]['value']) === OS2WEB_MEETINGS_CONTENT_TYPES_MEETING_DAGSORDEN) {`

Reply:
I think we can't use simply compare opreation, because we have also "dagsorden +" meeting type. But we can use constants like I did in this commit https://github.com/bellcom/os2dagsorden_profile/pull/4/commits/995178f01a4b729a38dfc30d22ebe1a25aa44bd5
or to use OR in the if statment
if ($meeting->field_os2web_meetings_type['und'][0]['value']) === OS2WEB_MEETINGS_CONTENT_TYPES_MEETING_DAGSORDEN || $meeting->field_os2web_meetings_type['und'][0]['value']) === OS2WEB_MEETINGS_CONTENT_TYPES_MEETING_DAGSORDEN_PLUS)

Reply:
yes, you are right.

let's keep it with strpos but change it to constant